docs/performance-guide.md

```diff
@@ -1,22 +1,51 @@
 Azure SignalR Service 1.0 Performance Guide
 ===========================================
 
+- [Terms](#terms)
+- [Audience](#audience)
+- [Overview](#overview)
+- [Performance insight](#perf_insight)
+  - [Performance factors](#perf_factors)
+  - [Find proper SKU](#evaluation)
+    - [Quick evaluation](#quick_evaluation)
+    - [Evaluation for mixed use cases](#mixed_evaluation)
+- [Case study](#case_study)
+  - [Default mode](#server)
+    - [Echo](#echo)
+    - [Broadcast message to all clients](#broadcast)
+    - [Broadcast message to a group](#sendToGroup)
+    - [Send message to a client](#sendToClient)
+    - [Evaluate perf for unknown scenario](#evaluatePerf)
+    - [AspNet SignalR Echo/Broadcast](#AspNetSignalR)
+  - [Serverless mode](#serverless)
+    - [Broadcast through REST API](#restBroadcast)
+    - [Send to user through REST API](#restSendToUser)
+- [Performance environment](#perf_env)
+- [Performance tools](#perf_tool)
+
+<a name="terms"></a>
 Terms
 -----
 
-ASRS: Azure SignalR Service
+*ASRS*: Azure SignalR Service
 
-Inbound: the incoming message to Azure SignalR Service
+*Inbound*: the incoming message to Azure SignalR Service
 
-Outbound: the outgoing message from Azure SignalR Service
+*Outbound*: the outgoing message from Azure SignalR Service
 
-Bandwidth: total size of all messages
+*Bandwidth*: total size of all messages
```

Comment:
Bandwidth is a rate, so should be total size of all messages sent in a time window (say a second)?

Reply:
Yes, you are right. I'll fix it.